bundles/EcommerceFrameworkBundle/CartManager/AbstractCart.php

```diff
@@ -950,11 +950,11 @@ protected function validateVoucherTokenReservations()
      */
     protected static function isValidCartItem(CartItemInterface $item)
     {
-        if ($item->getProduct() instanceof CheckoutableInterface) {
+        if ($item->getProduct()) {
```

Comment:
why remove that instanceof check?

Reply:
@fashxp getProduct returns CheckoutableInterface or null.

Reply:
good point. closed